remo/voting/templates/includes/voting_templates.html

```diff
@@ -1,167 +1,193 @@
-      <input name="${ prefix }-${ count }_radio_choices-MAX_NUM_FORMS"
-             value="1000"
-             id="id_${ prefix }-${ count }_radio_choices-MAX_NUM_FORMS"
-             type="hidden">
+      {% raw %}
+        <input name="{{ prefix }}-{{ count }}_radio_choices-TOTAL_FORMS"
+               value="1"
+               id="id_{{ prefix }}-{{ count }}_radio_choices-TOTAL_FORMS"
+               type="hidden">
+        <input name="{{ prefix }}-{{ count }}_radio_choices-INITIAL_FORMS"
+               value="0"
+               id="id_{{ prefix }}-{{ count }}_radio_choices-INITIAL_FORMS"
+               type="hidden">
+        <input name="{{ prefix }}-{{ count }}_radio_choices-MAX_NUM_FORMS"
+               value="1000"
+               id="id_{{ prefix }}-{{ count }}_radio_choices-MAX_NUM_FORMS"
+               type="hidden">
+      {% endraw %}
     </div>
   </div>
 </script>
 
-<script id="range-poll-tmpl" type="text/x-jquery-tmpl">
-  <input id="id_${ prefix }-${ count }-id" type="hidden" name="${ prefix }-${ count }-id">
+<script id="range-poll-tmpl" type="text/x-handlebars-template">
+{% raw %}<input id="id_{{ prefix }}-{{ count }}-id" type="hidden"
+                name="{{ prefix }}-{{ count }}-id">{% endraw %}
   <div class="copyblock voting-poll">
     <div class="poll-block">
       <div title="Range Poll Name" data-tooltip class="has-tip tip-left">
-        <input name="${ prefix }-${ count }-name"
-               placeholder="Range Poll Name"
-               maxlength="500"
-               id="id_${ prefix }-${ count }-name"
-               type="text">
+        {% raw %}
+          <input name="{{ prefix }}-{{ count }}-name"
+                 placeholder="Range Poll Name"
+                 maxlength="500"
+                 id="id_{{ prefix }}-{{ count }}-name"
+                 type="text">
+        {% endraw %}
       </div>
       <div class="required-field"></div>
-      <div class="formset" data-prefix="${ prefix }-${ count }_range_choices">
-        <input name="${ prefix }-${ count }_range_choices-0-id"
-               id="id_${ prefix }-${ count }_range_choices-0-id"
-               type="hidden">
+      <div class="formset" data-prefix="{{ prefix }}-{{ count }}_range_choices">
+        {% raw %}<input name="{{ prefix }}-{{ count }}_range_choices-0-id"
+                id="id_{{ prefix }}-{{ count }}_range_choices-0-id"
+                type="hidden">{% endraw %}
         <div class="copyblock nominee">
           <div class="select-nominee-group">
             <div title="Select Nominee"
-                 data-tooltip
-                 class="has-tip tip-left select-nominee">
-              <select id="id_${ prefix }-${ count }_range_choices-0-nominee"
-                      name="${ prefix }-${ count }_range_choices-0-nominee">
+                  data-tooltip
+                  class="has-tip tip-left select-nominee">
+              {% raw %}<select id="id_{{ prefix }}-{{ count }}_range_choices-0-nominee"
```

Comment:
While reviewing this, it seemed to me very unreadable. I don't have any strong opinions about this though.